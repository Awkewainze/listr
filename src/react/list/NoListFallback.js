import React from 'react';

const NoListFallback = () => (
  <div className="NoListFallback">
    <h3>This is unexpected.</h3>
    <p>We're sorry, we're not sure how you got here! No list could be loaded.</p>
  </div>
);

export default NoListFallback;