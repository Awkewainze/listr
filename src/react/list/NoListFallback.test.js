import React from 'react';
import { shallow } from 'enzyme';
import NoListFallback from './NoListFallback';

it('matches snapshot', () => {
  expect(shallow(<NoListFallback />)).toMatchSnapshot();
});