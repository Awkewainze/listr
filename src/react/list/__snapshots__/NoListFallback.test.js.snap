// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="NoListFallback"
>
  <h3>
    This is unexpected.
  </h3>
  <p>
    We're sorry, we're not sure how you got here! No list could be loaded.
  </p>
</div>
`;
